queries/tlaplus/locals.scm

```diff
@@ -0,0 +1,39 @@
+; Scopes
+[
+  (bounded_quantification)
+  (function_definition) 
+  (lambda) 
+  (module) 
+  (module_definition) 
+  (pcal_algorithm)
+  (pcal_macro)
+  (pcal_procedure)
+  (pcal_with)
+  (unbounded_quantification)
+] @scope
+
+; Definitions
+(constant_declaration (identifier) @definition.constant)
+(function_definition name: (identifier) @definition.function)
+(lambda (identifier) @definition.parameter)
+(operator_definition name: (identifier) @definition.function)
+(operator_definition parameter: (identifier) @definition.parameter)
+(pcal_macro_decl parameter: (identifier) @definition.parameter)
+(pcal_proc_var_decl (identifier) @definition.parameter)
+(pcal_var_decl (identifier) @definition.var)
+(pcal_with (identifier) @definition.parameter)
+(quantifier_bound (identifier) @definition.parameter)
+(quantifier_bound (tuple_of_identifiers (identifier) @definition.parameter))
+(variable_declaration (identifier) @definition.var)
+
```

Comment:
I think the locals are missing `@reference`

Reply:
Added `@reference`. For some reason, scope aware highlighting worked without it